experimenter/experimenter/experiments/tests/api/v5/test_mutations.py

```diff
@@ -1289,6 +1290,179 @@ def test_update_required_excluded_experiments_with_branches(self):
+                    "subscribers": [
+                        {
+                            "email": current_user.email,
+                            "subscribed": True,
+                        }
+                    ],
+                    "changelogMessage": "test subscribe",
+                }
+            },
+            headers={settings.OPENIDC_EMAIL_HEADER: current_user.email},
+        )
+        self.assertEqual(response.status_code, 200, response.content)
+        content = json.loads(response.content)
+        result = content["data"]["updateExperiment"]
+
+        self.assertEqual(result["message"], "success")
+
+        experiment = NimbusExperiment.objects.get()
+        self.assertEqual(list(experiment.subscribers.all()), [current_user])
+
+    def test_unsubscribe_to_experiment(self):
+        current_user = UserFactory.create()
+
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.LIVE_APPROVE_APPROVE,
+            subscribers=[current_user],
+        )
+        self.assertEqual(list(experiment.subscribers.all()), [current_user])
+
+        response = self.query(
+            UPDATE_EXPERIMENT_MUTATION,
+            variables={
+                "input": {
+                    "id": experiment.id,
+                    "subscribers": [
+                        {
+                            "email": current_user.email,
+                            "subscribed": False,
+                        }
+                    ],
+                    "changelogMessage": "test subscribe",
+                }
+            },
+            headers={settings.OPENIDC_EMAIL_HEADER: current_user.email},
+        )
+        self.assertEqual(response.status_code, 200, response.content)
+        content = json.loads(response.content)
+        result = content["data"]["updateExperiment"]
+
+        self.assertEqual(result["message"], "success")
+
+        experiment = NimbusExperiment.objects.get()
+        self.assertEqual(list(experiment.subscribers.all()), [])
+
+    def test_unsubscribe_to_experiment_when_not_subscribed(self):
+        current_user = UserFactory.create()
+
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.LIVE_APPROVE_APPROVE, subscribers=[]
+        )
+        self.assertEqual(list(experiment.subscribers.all()), [])
+
+        response = self.query(
+            UPDATE_EXPERIMENT_MUTATION,
+            variables={
+                "input": {
+                    "id": experiment.id,
+                    "subscribers": [
+                        {
+                            "email": current_user.email,
+                            "subscribed": False,
+                        }
+                    ],
+                    "changelogMessage": "test subscribe",
+                }
+            },
+            headers={settings.OPENIDC_EMAIL_HEADER: current_user.email},
+        )
```

Comment:
just a suggestion, you can fold all these tests into one and can use `parametrize`

Reply:
sounds good!